### PR TITLE
Prevent users from accidentally deploying entire filesystem.

### PR DESCRIPTION
**Pull request by TamalSaha**

### CODE REVIEW

File: lib/app.js [hunk 1]
```diff
@@ -342,6 +342,22 @@ function mkDirs(filePath) {
 
 module.exports = {
   init: function(argv) {
+    var curDir = process.cwd();
+    var homeDir = process.env[(process.platform == 'win32') ? 'USERPROFILE' : 'HOME'];
+    var isCurDirEmpty = fs.readdirSync(curDir).length === 0;
+    if (path.relative(homeDir, curDir).match(/^\./)) {
```
Comment (drtriumph) [inline lib/app.js]: Won't this also match the home directory?

File: lib/upload.js [hunk 2]
```diff
@@ -79,7 +80,13 @@ module.exports = {
                         'directory does not contain an index.html\n' +
                         'Make sure you\'re deploying the right public directory: ' +
                         chalk.bold(path.resolve(publicDir)));
+      } else if (fileCount > 500 || tarballSize > 50*1024*1024) {
+        console.log(chalk.yellow('Public Directory Warning') + ' - Uploading ' +
```
Comment (drtriumph) [inline lib/upload.js]: We use `WARNING` above, and `Warning` here - we should try and be consistent on case. I think lower-case is probably enough - depending on president considering it's already yellow

File: lib/upload.js [hunk 3]
```diff
@@ -79,7 +80,13 @@ module.exports = {
                         'directory does not contain an index.html\n' +
                         'Make sure you\'re deploying the right public directory: ' +
                         chalk.bold(path.resolve(publicDir)));
+      } else if (fileCount > 500 || tarballSize > 50*1024*1024) {
```
Comment (drtriumph) [inline lib/upload.js]: 50 MB isn't that big - perhaps we should increase this to 100?

**Comment by drtriumph** [comment]
Back to you with a few comments

**Comment by drtriumph** [comment]
The last two commits look good, but rather than waiting for the CI to finish - could you reopen this PR against master instead of the other branch

**Comment by TamalSaha** [comment]
Reopened pr against master #56

**Comment by drtriumph** [comment]
closing in favor of #56

**Pull request closed without merging**